Documentation/Testing/History.rst

```diff
@@ -248,7 +248,7 @@ this area, two further API's have been established: :ref:`Context <context-api>`
 :ref:`Environment <Environment>`. Remember each functional tests case runs in an own instance within
 typo3temp? TYPO3 Core  always had the PHP constant *PATH_site* that contained the path to the document
 root. With having test cases in different locations, this constant would have to change. But it
-can't, it's constant and PHP luckily does not allow redefining constants. The environment API
+can't, its constant and PHP luckily does not allow redefining constants. The environment API
```

Comment:
that should be "it is a constant", shouldn't it?

Reply:
Correct. I checked all replacements of "it's" again.